### PR TITLE
refactor(rds): update rds docs and add a rds instance testcase

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
update rds docs and add a rds instance testcase
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/rds" TESTARGS="-run TestAccRdsInstance_mariadb"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds -v -run TestAccRdsInstance_mariadb -timeout 360m -parallel 4
=== RUN   TestAccRdsInstance_mariadb
=== PAUSE TestAccRdsInstance_mariadb
=== CONT  TestAccRdsInstance_mariadb
--- PASS: TestAccRdsInstance_mariadb (448.95s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       448.988s

```

### CODE REVIEW

File: huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go [hunk 1]
```diff
@@ -931,6 +960,40 @@ resource "huaweicloud_rds_instance" "test" {
...
+  db {
+    password = "%[3]s"
+    type     = "MariaDB"
+    version  = "10.5"
+    port     = 3306
+  }
+
+  volume {
+    type     = "CLOUDSSD"
+    size     = 40
```
Comment (jinyangyang222) [inline huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go:991]: type = "CLOUDSSD"
    size = 40

Comment (profoundwu) [reply huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go]: fixed

File: docs/data-sources/rds_flavors.md [hunk 2]
```diff
@@ -33,10 +34,11 @@ type | version
 MySQL| 5.6 <br>5.7 <br>8.0
 PostgreSQL | 9.5 <br> 9.6 <br>10 <br>11 <br>12 <br>13
 SQLServer| 2008_R2_EE <br>2008_R2_WEB <br>2012_SE <br>2014_SE <br>2016_SE <br>2017_SE <br>2012_EE <br>2014_EE <br>2016_EE <br>2017_EE <br>2012_WEB <br>2014_WEB <br>2016_WEB <br>2017_WEB
+MariaDB| 10.5 <br>
```
Comment (ShiChangkuo) [inline docs/data-sources/rds_flavors.md]: `<br>` is used for multi versions

Comment (profoundwu) [reply docs/data-sources/rds_flavors.md]: fixed

**Pull request merged**